gensrc/proto/internal_service.proto

```diff
@@ -428,7 +428,7 @@ message PKafkaLoadInfo {
     required string brokers = 1;
     required string topic = 2;
     repeated PStringPair properties = 3;
-    required int64 warehouseId = 11;
+    optional int64 warehouseId = 11;
```

Comment:
I'm wondering if all set operations for this field are by `setWarehouseId()`.

Reply:
https://github.com/StarRocks/starrocks/blob/3aa2b039765d8d893b73b61edbf315aeac73be30/fe/fe-core/src/main/java/com/starrocks/common/util/KafkaUtil.java#L100-L117
All are set in `KafkaUtil::genPKafkaLoadInfo`

Reply:
should change `kafkaLoadInfo.warehouseId = warehouseId; ` into `kafkaLoadInfo.setWarehouseId(warehouseId); ` to make an option field valid. Right?